hub-detect/src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/clang/CompileCommandWrapper.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * hub-detect
+ *
+ * Copyright (C) 2018 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.hub.detect.bomtool.clang;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.synopsys.integration.util.Stringable;
+
+public class CompileCommandWrapper extends Stringable {
```

Comment:
I'm not sure I see the need for a wrapper. The getter methods on the Compile Command could do this. Or I'd prefer a word other than wrapper or this class. Something to consider.

Reply:
Here was my dilemma on that approach: The argument for public fields on JSON data objects was that that's how Gson treated them, so putting getters/setters in those classes mislead readers of the code into thinking they would be executed, when in fact, Gson goes around them. To stay true to that, I figured I'd better leave CompileCommand as-is (other than adding the field).

I agree on the name. Will try to think of a better one.